src/Feed/BrndFeedType.php

```diff
@@ -0,0 +1,183 @@
+
+        if ('' === $baseUri || null === $sportCenterId || '' === $sportCenterId) {
+            return $result;
+        }
+
+        $feedSource = $feed->getFeedSource();
+
+        if (null === $feedSource) {
+            return $result;
+        }
+
+        $bookings = $this->apiClient->getInfomonitorBookingsDetails($feedSource, $sportCenterId);
+
+        $result['bookings'] = array_map([$this, 'parseBrndBooking'], $bookings);
+
+        return $result;
+    }
+
+    private function parseBrndBooking(array $booking): array
+    {
+        // Parse start time
+        $startDateTime = null;
+        if (!empty($booking['dato']) && isset($booking['starttid']) && is_string($booking['starttid'])) {
+            // Trim starttid to 6 digits after dot for microseconds
+            $starttid = preg_replace('/\.(\d{6})\d+$/', '.$1', $booking['starttid']);
+            $dateOnly = substr($booking['dato'], 0, 10);
+            $dateTimeString = $dateOnly.' '.$starttid;
+            $startDateTime = \DateTimeImmutable::createFromFormat('m/d/Y H:i:s.u', $dateTimeString);
+        }
+
+        // Parse end time
+        $endDateTime = null;
+        if (!empty($booking['dato']) && isset($booking['sluttid']) && is_string($booking['sluttid'])) {
+            $sluttid = preg_replace('/\.(\d{6})\d+$/', '.$1', $booking['sluttid']);
+            $dateOnly = substr($booking['dato'], 0, 10);
+            $dateTimeString = $dateOnly.' '.$sluttid;
+            $endDateTime = \DateTimeImmutable::createFromFormat('m/d/Y H:i:s.u', $dateTimeString);
+        }
+
+        return [
+            'bookingcode' => $booking['ansøgning'] ?? '',
+            'remarks' => $booking['bemærkninger'] ?? '',
+            'startTime' => $startDateTime ? $startDateTime->getTimestamp() : null,
+            'endTime' => $endDateTime ? $endDateTime->getTimestamp() : null,
+            'complex' => $booking['anlæg'] ?? '',
+            'area' => $booking['område'] ?? '',
+            'facility' => $booking['facilitet'] ?? '',
+            'activity' => $booking['aktivitet'] ?? '',
+            'team' => $booking['hold'] ?? '',
+            'status' => $booking['status'] ?? '',
+            'checkIn' => $booking['checK_IN'] ?? '',
+            'bookingBy' => $booking['ansøgt_af'] ?? '',
+            'changingRooms' => $booking['omklædningsrum'] ?? '',
+        ];
+    }
+
+    public function getConfigOptions(Request $request, FeedSource $feedSource, string $name): ?array
+    {
+        return null;
+    }
+
+    public function getRequiredSecrets(): array
+    {
+        return [
+            'api_base_uri' => [
+                'type' => 'string',
+                'exposeValue' => true,
+            ],
+            'company_id' => [
+                'type' => 'string',
+                'exposeValue' => true,
+            ],
+            'api_auth_key' => [
+                'type' => 'string',
+                'exposeValue' => true,
```

Comment:
I think you should hide this value, otherwise everybody in the admin can see the value.

```suggestion
                'exposeValue' => false,
```

It will still be possible to set the value, it will just not be visible in the admin after being set.